packages/playground/src/components/Editor/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { useEffect, useState } from 'react';
+import MonacoEditor from 'react-monaco-editor';
+import exampleCode from './example-code';
+
+export const Editor = () => {
+  const [code, setCode] = useState(exampleCode);
+  const [isDark, setIsDark] = useState(true);
+  const [showMinimap, setShowMinimap] = useState(true);
+  const [showNumbering, setShowNumbering] = useState(true);
+
+  const handleChange = (newValue: string) => {
+    setCode(newValue);
+  };
+
+  const editorDidMount = (editor: any) => {
```

Comment:
I'd suggest to avoid `any` completely.

We can also add a rule for that (in the future, wouldn't do in this PR, I can create an issue)
https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-explicit-any.md